EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -149,7 +149,7 @@ as the action of a sparse kernel on the infections $I(t)$.
 ```math
 \begin{align}
 y_t &\sim \text{NegBinomial}(\mu = \sum_{s\geq 0} K[t, t-s] I(s), r), \\
-1 / r &\sim \text{Gamma}(3, 0.05/3).
+1 / \sqrt{r} &\sim \text{HalfNormal}\Big(0.1 \sqrt{{\pi \over 2}}\Big).
```

Comment:
we should probably give people a small hint as to the motivation for this

Reply:
I gave this a bit of write up in the `getting_started` note.